src/client/routes/main-router.jsx

```diff
@@ -1,14 +1,19 @@
 import React from "react";
 
-import { Route, withRouter, Link } from "react-router-dom";
+import { Route, withRouter, Link, Redirect } from "react-router-dom";
 
 import Home from "./../Home";
+import LoginForm from "./../components/login-form";
+import PrivateRoute from "./../components/PrivateRoute";
+import AppLayout from "./../components/AppLayout";
 
 const MainRouter = props => {
   return (
     <React.Fragment>
       <div className="container">
-        <Route exact path="/" component={Home} />
+        <Route exact path="/home" component={Home} />
```

Comment:
The path is `/` not `/home`. The component however is called `Home`.

Reply:
resolved the comment.